data/World/Spirit Temple.json

```diff
@@ -142,7 +172,9 @@
                 (is_adult and (Hookshot or Hover_Boots or logic_spirit_lobby_jump) and
                     ((Small_Key_Spirit_Temple, 3) or spirit_temple_shortcuts)) or
                 (logic_spirit_lobby_gs and Boomerang and (Hookshot or Hover_Boots or logic_spirit_lobby_jump) and
-                    (has_explosives or ((Small_Key_Spirit_Temple, 2) and bombchus_in_logic and not entrance_shuffle)))"
+                    (has_explosives or ((Small_Key_Spirit_Temple, 2) and bombchus_in_logic and not entrance_shuffle)))",
+            "Spirit Temple Central Chamber Flying Pot 1" : "True",
+            "Spirit Temple Central Chamber Flying Pot 2" : "True" 
```

Comment:
The logic for these pots needs to match with the hall to the iron knuckle pots. Neither age needs any items to traverse between the two areas.